packages/common/components/content/blocks/page-header.marko

```diff
@@ -1,11 +1,28 @@
 import { getAsObject } from '@base-cms/object-path';
 
 $ const content = getAsObject(input, 'content');
+$ const noPublishDateContentTypes = ['event', 'webinar', 'whitepaper', 'contact'];
```

Comment:
@brandonbk and I discussed this - whitepapers should still display the published date. As such, could you remove it from this list?

Reply:
Okay, just let me know what I should tell Deb in regards to that.

Reply:
Okay, just let me know what I should tell Deb in regards to that.

Reply:
Just let her know that BK and I discussed it. It's needed for existing websites